packages/core-plugin-api/api-report.md

```diff
@@ -692,6 +692,19 @@ export enum SessionState {
   SignedOut = 'SignedOut',
 }
 
+// @public
+export interface SignalsApi {
+  subscribe(
+    pluginId: string,
+    onMessage: (data: unknown) => void,
+    topic?: string,
+  ): void;
+  unsubscribe(pluginId: string, topic?: string): void;
+}
+
+// @public
+export const signalsApiRef: ApiRef<SignalsApi>;
```

Comment:
I'm thinking that we should at least for now add this outside of the frontend core API, since that's 1.0 and needs to remain stable. One option is to ship it under an `/alpha` export but tbh I'd rather have it in a separate package to begin with. If we want to move it into core in the future once stable that's going to be fairly straightforward.